### ISSUE TITLE
placeholderOption must set empty value

### ISSUE THREAD

**Issue by po-ku** [LaravelCollective/html#65]
if not set a value attribute then submit a inner tag.

```PHP
 < return '<option' . $this->html->attributes($options) . '>' . e($display) . '</option>';
 ---
 > return '<option' . $this->html->attributes($options) . ' value="">' . e($display) . '</option>';
```

**Comment by adamgoose** [comment]
This has already been fixed in #63. Run a `composer update` to get the latest version!

**Comment by po-ku** [comment]
Thx so much. (y)
